grub.cfg-normal

```diff
@@ -31,12 +31,11 @@ elif [ "$snap_mode" = "trying" ]; then
     save_env snap_mode
 fi
 
-set label="writable"
+set label="ubuntu-data"
 set cmdline="root=LABEL=$label snap_core=$snap_core snap_kernel=$snap_kernel ro net.ifnames=0 init=/lib/systemd/systemd console=ttyS0 console=tty1 panic=-1"
 
 menuentry "$snap_menuentry" {
-    search --label $label --set=writable
-    loopback loop ($writable)/system-data/var/lib/snapd/snaps/$snap_kernel
-    linux (loop)/kernel.img $cmdline
-    initrd (loop)/initrd.img
+    search --label ubuntu-boot --set=ubuntu_boot
+    linux ($ubuntu_boot)/kernel.img $cmdline systemd.debug-shell=1
```

Comment:
'systemd.debug-shell=1' 
This looks like an unrelated change that doesn't belong on the actual core20 branch.

Reply:
This is there to make debugging easier, if you feel strongly about it we could remove it. Note that this is the "uc20" spike branch we are merging into - this is not a "production" branch or anything like this.

Reply:
How will we make sure we clean up all debugging-only code when we go to production?  Spike or not, that's what worries me since this doesn't interfere with the booting of the system but does leave a security hole around.

Reply:
As agreed before starting the spike, this branch won't be merged into master -- it will serve only as a reference and all changes will be properly rewritten at a later moment, which should serve as a filter to remove debugging code. That said, most early problems related to snapd seeding have already been fixed so I believe we don't need the debug shell anymore. I'll push a change to remove it.